samples/highcharts/demo/bar-force/demo.html

```diff
@@ -0,0 +1,30 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<figure class="highcharts-figure">
+    <div id="container" aria-label="Stacked bar 'force' chart"></div>
+    <p class="highcharts-description">
+        This demo shows a horizontally stacked bar chart to show how
+        various forces contribute to a specific outcome. Each force is depicted
+        by a separate bar, stacked together to visually represent their combined
+        effect. This visualization aids in understanding the interaction between
+        the forces involved, like identifying imbalances and/or each force's
+        significance within that specific outcome.<br/>
+        <br/>
```

Comment:
Use nested paragraphs instead of double breaks.

Reply:
Oh wait a minute; that's not valid HTML. Instead, make the `.highcharts-description` a div, then paragraphs inside.